### PR TITLE
Forms component

### PR DESCRIPTION
**Pull request by prath-dev**
Added Forms Components .
Live Preview :http://127.0.0.1:5501/index.html#forms

### CODE REVIEW

**Comment by rohitrout** [comment]
looks good to go, apply some padding

File: components/Forms/form.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
+.login-container {
+    height: 90%;
+    display: flex;
+    justify-content: center;
+    /* margin-top: 7rem;*/
```
Comment (kaustubh-jsr) [inline components/Forms/form.css:5]: You can remove commented code.

**Review by kaustubh-jsr — COMMENTED** [review]
Code is good. You can give some margins to the input components. Currently they are touching each other.

**Review by prathameshpcodes — APPROVED** [review]
try giving margin & padding to the components , also remove unwanted commented code

**Pull request merged**